pu.py

```diff
@@ -25,9 +25,14 @@ def __init__(self, period):
             puWeightProducer.initialized = True
 
     def getWeight(self, df):
```

Comment:
add `return_variations` (same logic as for tau)

Reply:
ok